lib/matplotlib/testing/__init__.py

```diff
@@ -8,6 +8,8 @@
 import matplotlib
 from matplotlib.cbook import is_string_like, iterable
 from matplotlib import rcParams, rcdefaults, use
+# For backward compatibility
+from . import noseclasses
```

Comment:
This line would always raise a deprecationwarning, I guess, and is not needed for backcompat anyways as noseclasses was not directly imported anyways?